tests/integration/test_fabric_extractor.py

```diff
@@ -89,6 +94,27 @@ def raw_time_series(request, azure_credential, cognite_client, test_extractor):
     )
 
 
+@pytest.fixture(scope="function")
+def test_csv_file_path(test_extractor, azure_credential, cognite_client):
+    os.makedirs(TEST_FILE_DIRECTORY, exist_ok=True)
+
+    file_path = os.path.join(TEST_FILE_DIRECTORY, TEST_FILE_NAME)
+    with open(file_path, "w") as file:
+        file.write("Test data")
```

Comment:
Why not use an existing file from the repo (like movie_model.graphql) rather than create your own file dynamically? Not a huge issue, but seems like unnecessary steps.

Reply:
I would rather have a file for the sole purpose of the fabric file -> cdf file scenario rather than reuse files for multiple things, that way if we see it in Fabric or CDF we know where it came from.  I can rename the file to be more specific to this test.